// Copyright 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for the article.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;
option retain_unknown_fields = true;

package sync_pb;

// Properties of Article objects.
message ArticleSpecifics {
  // Next ID to use: 4

  optional string entry_id = 1;

  optional string title = 2;

  repeated ArticlePage pages = 3;
}

message ArticlePage {
  // Next ID to use: 2

  optional string url = 1;
}
